wrappers/python/examples/wnt/README.md

```diff
@@ -0,0 +1,64 @@
+# WNT API Examples
+
+This folder contains the WNT API Examples.
+
+## Installation
```

Comment:
Change the title to requirements, since we are not installing the examples but instead making use of the messaging package.

Reply:
Done